Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -4148,17 +4157,63 @@ public boolean isInternetAvailable() {
 
     @JavascriptInterface
     public void showDialer(String phoneNum, boolean call) {
-        Intent intent = new Intent(call ? Intent.ACTION_CALL : Intent.ACTION_DIAL);
-        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-        intent.setData(Uri.parse("tel:" + phoneNum));
-        if (call && ContextCompat.checkSelfPermission(bridgeComponents.getContext(), Manifest.permission.CALL_PHONE) != PackageManager.PERMISSION_GRANTED) {
-            phoneNumber = phoneNum;
-            ActivityCompat.requestPermissions(bridgeComponents.getActivity(), new String[]{Manifest.permission.CALL_PHONE}, REQUEST_CALL);
-        } else {
-            bridgeComponents.getContext().startActivity(intent);
+        System.out.println("Inside show dialer function -------------------------------------------------------------------------------------------");
+        showDialerVoip(phoneNum, call);
+//        Intent intent = new Intent(call ? Intent.ACTION_CALL : Intent.ACTION_DIAL);
+//        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+//        intent.setData(Uri.parse("tel:" + phoneNum));
+//        if (call && ContextCompat.checkSelfPermission(bridgeComponents.getContext(), Manifest.permission.CALL_PHONE) != PackageManager.PERMISSION_GRANTED) {
+//            phoneNumber = phoneNum;
+//            ActivityCompat.requestPermissions(bridgeComponents.getActivity(), new String[]{Manifest.permission.CALL_PHONE}, REQUEST_CALL);
+//        } else {
+//            bridgeComponents.getContext().startActivity(intent);
+//        }
```

Comment:
Can this commented code be removed?

Reply:
sure, we are still testing the integration, will be making a commit with better changes soon